docs/source/maskmaking/index.rst

```diff
@@ -120,6 +120,7 @@ Here are some resources that either provide masks or tools for you to make your
 
 - `DeepWaterMap  <https://github.com/isikdogan/deepwatermap>`_ is a trained deep convolutional neural network that you can apply to Landsat/Sentinel multispectral imagery to create your own masks. You can also improve DeepWaterMap's base model by adding more training data. Requires some knowledge of Tensorflow.
 
+- A recent development as of April 2023 is the release of the *Segment Anything Model* `(SAM) <https://segment-anything.com/>`_ from Meta Research which allows rapid segmentation of any image, including satellite imagery. The SAM model has an API in Python via `segment-geospatial <https://github.com/opengeos/segment-geospatial>`_. *segment-geospatial* and could be used for rapid identification of water from various satellite imagery sources, e.g. Planet or Landsat.
```

Comment:
Is there a superfluous "and" after *segment-geospatial*?

"*segment-geospatial* and could..."

Reply:
I usually build the documentation locally. Just checked now and it doesn't look like we actually document how to do that 😬. The shell commands from the repository root are:

```
cd docs
make docs
make html
```

which is what we do for the actual automated deployment ([L39](https://github.com/VeinsOfTheEarth/RivGraph/blob/38f2a252a8f64ec88368d162b9794e1e5ff02b4c/.github/workflows/docs.yml#L39)), but we should definitely document the process for installing the required libraries and doing that local build.